media/css/pebbles/base/elements/_document.scss

```diff
@@ -6,12 +6,12 @@
 
 
 html {
-    font-size: 100%;
+    font-size: 18px;
```

Comment:
Been thinking about this a bit more and doing some testing. If we declare `18px` explicitly on both `html` and `body` it will override user preferences. But if we set `html` to `112.5%` (which comes out to 18px for the vast majority who don't change the 16px default) then use the font-size mixin on `body` (`@include font-size(18px);`) we should get the overall embiggening while still respecting the user's settings. Thoughts?